packages/mwp-app-render/src/components/WriteStyles.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+
+/**
+ * @module WriteStyles
+ *
+ * Adds style module CSS to `head` for both
+ * server and client render via `Helmet`.
+ *
+ * Usage:
+ * ```
+ * import WriteStyles from 'mwp-app-render/lib/components/WriteStyles';
+ * import styles from './myComponent.module.scss';
+ *
+ * const MyComponent = (props) => {
+ *		return (
+ *			<div>
+	 *			<WriteStyles styles={styles} />
+	 *			<p className={styles.someClass}>Hello world!</p>
+ *			</div>
+ *		);
+ * };
+ * ```
+ */
+const WriteStyles = props => {
+	const moduleCSS = new Set();
+	props.styles.forEach(style => {
+		moduleCSS.add(style._getCss()); // eslint-disable-line no-underscore-dangle
+	});
+
+	return (
+		<Helmet defer={false}>
+			<style type="text/css">
+				{[...moduleCSS].join('')}
```

Comment:
I _think_ it would be better to output this with `dangerouslySetInnerHtml` so that valid CSS characters like `>` don't get HTML-escaped.